Library/Homebrew/extend/ARGV.rb

```diff
@@ -59,8 +59,8 @@ def kegs
 
       raise NoSuchKegError, rack.basename if dirs.empty?
 
-      linked_keg_ref = HOMEBREW_LIBRARY.join("LinkedKegs", rack.basename)
-      opt_prefix = HOMEBREW_PREFIX.join("opt", rack.basename)
+      linked_keg_ref = HOMEBREW_LINKED_KEGS/rack.basename
+      opt_prefix = HOMEBREW_PREFIX/"opt/#{rack.basename}"
```

Comment:
It would be nice to have a `HOMEBREW_OPT_PREFIX` too if it appears often enough. I didn't check.